### ISSUE TITLE
Endianness flag doesn't affect data

### ISSUE THREAD

**Issue by easmith5** [kratsg/ironman#15]
Response packet from an IPBus packet requesting read of a register address:

```
Container:
    pointer = 32
    bigendian = False
    header = Container:
        protocol_version = 2
        reserved = 0
        id = 0
        byteorder = 15
        type_id = (enum) CONTROL 0
    transactions = ListContainer:
        Container:
            header = Container:
                protocol_version = 2
                id = 0
                words = 1
                type_id = (enum) READ 0
                info_code = (enum) SUCCESS 0
            address = 2684420096
            data = ListContainer:
                277440421
    status = None
    resend = None
```

But despite ```bigendian = False```, the data returned is 277440421 or '0x108967a5' in hex which is flipped from the actual value of '0xA5678910.

**Comment by kratsg** [comment]
I think I have it solved in a branch now:

```
>>> in_data = '200000f020000100deadbeef'.decode('hex')
>>> packet = IPBusConstruct.parse(data)
>>> print(packet)
Container: 
    pointer = 240
    bigendian = True
    header = Container: 
        protocol_version = 2
        reserved = 0
        id = 0
        byteorder = 15
        type_id = (enum) CONTROL 0
    transactions = ListContainer: 
        Container: 
            header = Container: 
                protocol_version = 2
                id = 0
                words = 1
                type_id = (enum) READ 0
                info_code = (enum) SUCCESS 0
            address = None
            data = ListContainer: 
                3735928559
    status = None
    resend = None
>>> packet.pointer=0x0
>>> IPBusConstruct.build(packet).encode('hex')
'f000002000010020efbeadde'
```

This works correctly for me now, I think. Is this what you expect?

**Comment by easmith5** [comment]
It looks like it did. Does the `packet.pointer=0x0 ` do the same thing as byteswap?

**Comment by easmith5** [comment]
Nevermind, it's still backwards if I don't return data[::-1] in the read method

**Comment by kratsg** [comment]
Closed with #18.
